test/components/common/stat/Stat.test.js

```diff
@@ -570,6 +570,14 @@ describe('Stat', () => {
       expect(inputGroup().props().onChange).to.be.a.function;
       expect(inputGroup().props().onSuffixChange).to.be.a.function;
     });
+
+    it('should should call the `onInputChange` Stat prop when provided on input change', () => {
+
```

Comment:
Bit of an echo in here. I think these tests might have been glanced over after being stubbed.

Reply:
I filled in the test stub for assigning default input and suffix values from props, and deleted the test stub for calling onInputChange, which is actually handled in the `propagateInputChange` tests, and the `handleInputChange` and `handleSuffixChange` tests that assert that it's called.